layout/paddedlayout_test.go

```diff
@@ -0,0 +1,51 @@
+package layout_test
+
+import (
+	"image/color"
+	"testing"
+
+	"fyne.io/fyne"
+	"fyne.io/fyne/canvas"
+	"fyne.io/fyne/layout"
+	"fyne.io/fyne/theme"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestPaddedLayout(t *testing.T) {
+	size := fyne.NewSize(100, 100)
+
+	obj := canvas.NewRectangle(color.NRGBA{0, 0, 0, 0})
+	container := &fyne.Container{
+		Objects: []fyne.CanvasObject{obj},
+	}
+	container.Resize(size)
+
+	layout.NewPaddedLayout().Layout(container.Objects, size)
+
+	assert.Equal(t, obj.Size().Width, size.Width-2*theme.Padding())
+	assert.Equal(t, obj.Size().Height, size.Height-2*theme.Padding())
+}
+
+func TestPaddedLayout_MinSize(t *testing.T) {
+	text := canvas.NewText("FooBar", color.NRGBA{0, 0xff, 0, 0})
```

Comment:
if we're not going to see it I'd be tempted to use color.Black or color.White...

Reply:
fixed